### PR TITLE
Changelog: Rephrase to move related entities closer together.

### PR DESCRIPTION
**Pull request by antkmsft**
I also briefly reviewed commits, (added Context-related changes), everything here should be up-to-date now.

These two PRs needs to be completed before this one:
- https://github.com/Azure/azure-sdk-for-cpp/pull/1879 [Merged, changelog integrated]
- https://github.com/Azure/azure-sdk-for-cpp/pull/1875 [+ merge the changelog entry as "- Changed return type of `Operation<T>::Poll()` from `std::unique_ptr<RawResponse>` to `RawResponse const&`."]

Then, we need to merge with master, and integrate changelog entries here.

Then we need to put release date - here and in Identity. And only after that check it in and start the release.

### CODE REVIEW

**Comment by antkmsft** [comment]
Gonna make more changes here, per team's ask

**Review by RickWinter — CHANGES_REQUESTED** [review]
This draft is wrong.  The RawResponse() method is new this month

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -61,6 +60,7 @@
 
 - Added support for HTTP conditional requests `MatchConditions` and `RequestConditions`.
 - Added the `Hash` base class and MD5 hashing APIs to the `Azure::Core::Cryptography` namespace available from `azure/core/cryptography/hash.hpp`.
+- Added `Azure::Core::Operation<T>::GetRawResponse()`.
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:63]: This belongs as a new feature for beta.7.

**Review by vhvb1989 — APPROVED** [review]

**Review by RickWinter — APPROVED** [review]

**Comment by ahsonkhan** [comment]
Here are the source changes between beta.6 and the upcoming beta.7 release from master:
https://github.com/Azure/azure-sdk-for-cpp/compare/2d4d766c6d454bfbd2a3627c5a3d8a4d81196480...6a8aff701c825d68c8b444c4948e7ca1eb719aa2

**Comment by ahsonkhan** [comment]
bug fix: https://github.com/Azure/azure-sdk-for-cpp/pull/1825

**Review by vhvb1989 — APPROVED** [review]

**Review by ahsonkhan — APPROVED** [review]
Looks good.

**Comment by antkmsft** [comment]
TODO: add
```
- Changed return type of `Operation<T>::Poll()` from `std::unique_ptr<RawResponse>` to `RawResponse const&`.
```

**Comment by ahsonkhan** [comment]
For future release changelogs, we should acknowledge community contributions.

For example, a section like this: https://github.com/Azure/azure-sdk-for-cpp/pull/1593

### Acknowledgments

Thank you to our developer community members who helped to make Azure Core better with their contributions to this release:

- Sylvain Joubert _([GitHub](https://github.com/sjoubert))_

### Bug Fixes

- Fixed computation of the token expiration time in `BearerTokenAuthenticationPolicy`. (A community contribution, courtesy of _[sjoubert](https://github.com/sjoubert))_

**Comment by antkmsft** [comment]
@ahsonkhan Other PRs: https://github.com/Azure/azure-sdk-for-cpp/pull/1593https://github.com/Azure/azure-sdk-for-cpp/pull/1522https://github.com/Azure/azure-sdk-for-cpp/pull/1239https://github.com/Azure/azure-sdk-for-cpp/pull/981
Fix math for bearer token refresh time by sjoubert · Pull Request #1593 · Azure/azure-sdk-for-cpp
Pull Request Checklist
Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:
See the detai...

**Pull request merged**